media/css/externalpages/pocket/components/_platform-footer.scss

```diff
@@ -0,0 +1,124 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+@import '~@mozilla-protocol/core/protocol/css/includes/lib';
+
+.platform-footer {
+    border-top: 1px solid #cacaca;
+    background-color: $color-light-gray-05;
+    font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+    padding-top: 32px;
+
+    &__navigation {
+        display: flex;
+        flex-direction: column-reverse;
+        margin: 0 auto;
+        max-width: 1000px;
+        width: 100%;
+
+        @media #{$mq-md} {
+            flex-direction: row;
+            justify-content: space-between;
+        }
+
+        &-links {
+            display: flex;
+            flex-direction: column;
+            font-size: 12px;
+            font-weight: 700;
+            padding: 0 16px;
+            margin: 0;
+
+            &-item {
+                margin: 0 0 16px;
+                cursor: pointer;
+
+                a:hover {
+                    text-decoration: underline;
+                }
+            }
+
+            @media #{$mq-md} {
+                flex-direction: row;
+
+                &-item {
+                    margin: 0 10px 0;
+
+                }
+            }
+
+            @media #{$mq-lg} {
+                margin-right: 21px;
+            }
+        }
+
+        &-social-media {
+            padding: 0 16px;
+        }
+
+        &-social-media-list {
+            margin: 0;
+            padding: 0;
+            display: flex;
+
+            &-text {
+                font-size: 12px;
+                font-weight: 700;
+                margin-right: 16px;
+            }
+
+            &-item {
+                background: url('/media/img/externalpages/pocket/footer-socialicons-high-res.png') 0 1px no-repeat;
```

Comment:
Instead of using a background for social icons, could we possibly just borrow social icons from something like Font Awesome? I see that you're using background sizing, positioning, etc, and that is probably an over-kill just to display the icons. I would say to just borrow the icons, and place `aria-labels` in the element holding them for better accessibility.